### PR TITLE
[DEV-11261] rename daims to gsdm

### PR DESCRIPTION
**Pull request by sukhusaspending**
**Description:**
Rename “DATA Act Broker Submission” to “Data Accountability Broker Submission” (DABS) to align with the rebranding of DAIMS to GSDM. This spans updates within the Broker, GSDM documents, and references on USAspending, ensuring consistency across all materials for the Annual Release GSDM v1.1 in November. 

**Technical details:**
The technical details for the knowledge of other developers. Any detailed caveats or specific deployment steps should be outlined here.

**Requirements for PR merge:**

1. [X] Unit & integration tests updated
2. [X] API documentation updated
3. [X] Necessary PR reviewers:
    - [X] Backend
4. [X] Jira Ticket [DEV-11261](https://federal-spending-transparency.atlassian.net/browse/DEV-11261):
    - [X] Link to this Pull-Request
    

**Area for explaining above N/A when needed:**
```
```

### CODE REVIEW

File: usaspending_api/data/daims_maps.py
Comment (aguest-kc) [inline usaspending_api/data/daims_maps.py:1]: Can you also change the name of this file from `daims_maps.py` to `gsdm_maps.py`? It doesn't look like this file is used in any imports so it shouldn't break anything.

**Review by aguest-kc — CHANGES_REQUESTED** [review]

File: usaspending_api/data/daims_maps.py
Comment (sukhusaspending) [reply usaspending_api/data/daims_maps.py:1]: I renamed daims_maps.py to gsdm_maps.py. Please review again

**Review by aguest-kc — APPROVED** [review]

**Review by dpb-bah — CHANGES_REQUESTED** [review]
Could you also update the README? Under `Seeding and Loading Database Data`

```
- For details on loading reference data, DATA Act Broker submissions, and current USAspending data into the API, see [loading_data.md](loading_data.md).
```

`DATA Act Broker submission` -> `Data Accountability Broker Submission`

File: README.md [hunk 1]
```diff
@@ -110,7 +110,7 @@ _Alternatively, to download a fully populuated production snapshot of the databa
 
 _**Executing individual data-loaders** to load in data is also possible, but requires more familiarity with those ad-hoc scripts and commands, and also requires an external data source (DATA Broker DB, or external file, etc.) from which to load the data._
 
-- For details on loading reference data, DATA Act Broker submissions, and current USAspending data into the API, see [loading_data.md](loading_data.md).
+- For details on loading reference data, Data Accountability Broker Submission, and current USAspending data into the API, see [loading_data.md](loading_data.md).
```
Comment (dpb-bah) [inline README.md:113]:
```suggestion
- For details on loading reference data, Data Accountability Broker Submissions, and current USAspending data into the API, see [loading_data.md](loading_data.md).
```

Comment (dpb-bah) [reply README.md:113]: Adding an `s` to keep it plural and consistent.

**Review by dpb-bah — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (sukhusaspending) [reply README.md]: thanks and done

**Review by dpb-bah — APPROVED** [review]

**Pull request merged**